tests/unit/test_table_service.py

```diff
@@ -145,7 +145,7 @@ def test_insert_full_overwrite(
         ("aliased", "TEXT"),
     ]
 
-    mocker.patch("firebolt_ingest.table_service.raise_on_tables_non_compatibility")
+    # mocker.patch("firebolt_ingest.table_service.raise_on_tables_non_compatibility")
```

Comment:
Do we need to comment out mocks here and in other place? I believe they should still work well

Reply:
it fails on "does not have the attribute 'raise_on_tables_non_compatibility'" which is true, because there is no raise_on_tables_non_compatibility in the function, so no object for mocker